docs/kvm/consumer-pod.yml

```diff
@@ -0,0 +1,12 @@
+apiVersion: v1
+kind: Pod
+metadata:
+  name: kvm-consumer
+spec:
+  containers:
+  - name: busybox
+    image: busybox
+    command: ["/bin/sleep", "123"]
+    resources:
+      limits:
+        devices.kubevirt.io/kvm: 1
```

Comment:
Didn't the consumer previously had to specify limits==requests==n?

Reply:
No, limits is enough, as this will set requests to the same value by default.